packages/target-ethers-v5/src/index.ts

```diff
@@ -129,6 +133,20 @@ export default class Ethers extends TsGeneratorPlugin {
     return allFiles
   }
 
+  private genCommons(): string {
+    return `
+    import { EventFilter, Event } from 'ethers'
+    import { Result } from '@ethersproject/abi'
+
+    export interface TypedEventFilter<T, G> extends EventFilter {}
```

Comment:
why are the `T` & `G` args here if they're not used?

Reply:
Actually the `T` & `G` contain type info utilised by `queryFilter` and other methods for output